src/ApparentHorizons/StrahlkorperGr.cpp

```diff
@@ -202,20 +202,22 @@ Scalar<DataVector> spin_function(
       make_with_value<Scalar<DataVector>>(area_element, 0.0);
   for (size_t i = 0; i < 3; ++i) {
     for (size_t j = 0; j < 3; ++j) {
+      const DataVector extrinsic_curvature_normal_sin_theta =
+          extrinsic_curvature.get(i, j) * unit_normal_vector.get(j) *
+          get(sin_theta);
+
```

Comment:
close the j loop

Reply:
If I close the j loop here, then my DataVector is out of scope before I can multiply it into extrinsic_curvature_theta_normal_sin_theta and extrinsic_curvature_phi_normal. Also, that will give an incorrect result for the summation, won't it?

Reply:
declare the DataVector in the i loop, initialized to zero